x-pack/plugins/fleet/server/services/telemetry/fleet_usages_schema.ts

```diff
@@ -201,18 +189,18 @@ export const fleetUsagesSchema: RootSchema<any> = {
       },
     },
   },
-  agent_logs_top_errors: {
-    type: 'array',
-    items: {
-      type: 'text',
-      _meta: { description: 'Top messages from agent error logs' },
-    },
-  },
-  fleet_server_logs_top_errors: {
-    type: 'array',
-    items: {
-      type: 'text',
-      _meta: { description: 'Top messages from fleet server error logs' },
-    },
-  },
+  // agent_logs_top_errors: {
+  //   type: 'array',
+  //   items: {
+  //     type: 'text',
+  //     _meta: { description: 'Top messages from agent error logs' },
+  //   },
+  // },
+  // fleet_server_logs_top_errors: {
+  //   type: 'array',
+  //   items: {
+  //     type: 'text',
+  //     _meta: { description: 'Top messages from fleet server error logs' },
+  //   },
+  // },
```

Comment:
Let's delete this instead of commenting. We have git history to help us restore this code if we need it.